test/testdefs.jl

```diff
@@ -2,7 +2,7 @@
 
 using Test
 
-function runtests(name, isolate=true; seed=nothing)
+function runtests(name, path, isolate=true; seed=nothing)
```

Comment:
As this file is `include`'d, doesn't this function know about `test_path` ? (to avoid passing the `path` argument here)

Reply:
I think I prefer to keep it explicit like this to potentially ease the use of this file in other contexts.

Reply:
Ok no problem. But maybe then as in most of the cases the name is equal to the path, there could be a default, or even better a keyword `path=name`. But also I find it tricky that the path is still not the real path, as ".jl" has to be added. I guess this logic comes from the use of `name` before, but I see no reason now that `path` should not contain the .jl extension. Then the default value for `path` would be `name * ".jl"`.